site/content/docs/terraform-provider/checks-groups.md

```diff
@@ -31,26 +31,20 @@ resource "checkly_check" "e2e-login" {
     "eu-central-1"
   ]
 
-  script = <<EOT                              // The script the check should execute
-const { chromium } = require("playwright");
+  // The script the check should execute
```

Comment:
I moved the comment up because it's causing some formatting issues (see [live marketing page](https://www.checklyhq.com/docs/terraform-provider/checks-groups/#browser-checks))
<img width="921" alt="Screenshot 2023-02-22 at 10 24 32" src="https://user-images.githubusercontent.com/54396648/220578500-839020b6-091d-442b-88b9-ac145ad34506.png">